.kokoro/docker/Dockerfile

```diff
@@ -87,6 +87,17 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/* \
   && rm -f /var/cache/apt/archives/*.deb
 
+# Install Bazel for compiling Tink in Cloud SQL Client Side Encryption Samples
```

Comment:
LGTM. Could you add a link to https://github.com/google/tink/issues/483 so this can (hopefully) be removed in the future?

Reply:
👍